packages/algolia/schema/crn/algolia-schema.json

```diff
@@ -63,7 +64,8 @@
     "interestGroup.id",
     "workingGroup.id",
     "workingGroups.id",
-    "_tags"
+    "_tags",
+    "source"
```

Comment:
Source? Dont we have meta data saying what type it is?

Reply:
We don't have it. We check workingGroups to see if it is a working groups RO or a team RO https://github.com/yldio/asap-hub/blob/master/packages/react-components/src/organisms/SharedResearchCard.tsx#L60